src/m365/util/commands/accesstoken/accesstoken-get.ts

```diff
@@ -65,7 +70,20 @@ class UtilAccessTokenGetCommand extends Command {
 
     try {
       const accessToken: string = await auth.ensureAccessToken(resource, logger, this.debug, args.options.new);
-      await logger.log(accessToken);
+
+      if (args.options.decoded) {
```

Comment:
Could you move this to `accessTokenUtil.ts`, it seems to me that's a better place for it...